farcy/handlers.py

```diff
@@ -2,11 +2,13 @@
 
 from __future__ import print_function
 from collections import defaultdict
+from .const import CONFIG_DIR
```

Comment:
Can you move this to just above `.exceptions`? I think the general practice is to do system imports prior to local imports.